packages/components/app/styles/components/link/inline.scss

```diff
@@ -0,0 +1,66 @@
+//
+// LINK > INLINE COMPONENT
+//
+// properties within each class are sorted alphabetically
+// notice: pseudo-classes for the states *must* follow the order link > visited > focus > hover > active
+// see https://github.com/hashicorp/design-system-components/issues/132
+//
+//
+
+.hds-link-inline {
+  border-radius: 2px; // this is used by the outline too
+
+  &:focus,
+  &.is-focus,
+  &:focus-visible {
+    outline: 2px solid var(--token-color-focus-action-internal);
+    outline-offset: 1px;
+    // text-decoration: none; // TBD with CVeight
```

Comment:
Leaving a comment only to make sure we don't forget about this line

Reply:
fixed.